docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -436,10 +436,10 @@ The probability of missing a follow-up appointment is 8.68% for all simulants. [
     - Notes
   * - A
     - ASCVD = -19.5 + (0.043 * SBP) + (0.266 * Age) + (2.32 * Sex) where Sex=1 for males and Sex=0 for females 
-    - 
+    - This equation returns percentage values. It is okay if they are negative. 
   * - B
-    - LDL-C measreument error pulled from a normal distribution with mean=0 and SD=0.08 mmol/L    
-    - [McCormack_2020]_
+    - LDL-C measreument error pulled from a normal distribution with mean=0 and SD=0.08 mmol/L 
+    - Measurements have a minimum value of 0 enforced [McCormack_2020]_
```

Comment:
We should have the analogous note for sbp (minimum of 50 enforced)

Reply:
Okay, I was thinking we could do it just for LDL-C since the negatives are a bigger issue, but I added to SBP as well!